src/item/ItemFactory.ts

```diff
@@ -0,0 +1,450 @@
+import { Rarity, WearableRepresentation } from '@dcl/schemas'
+import {
+  computeHashes,
+  prefixContentName,
+  sortContent
+} from '../content/content'
+import { Content, RawContent, SortedContent } from '../content/types'
+import { THUMBNAIL_PATH } from './constants'
+import {
+  BodyShapeType,
+  BuiltItem,
+  ItemType,
+  LocalItem,
+  ModelMetrics,
+  WearableBodyShape,
+  WearableCategory
+} from './types'
+
+export class ItemFactory<X extends Content> {
```

Comment:
This is a bit personal, but what do you think about moving private methods after the public ones? I feel like you shouldn't want to see those but it's cumbersome to have to scroll thru a class trying to find the methods that you actually care about